### PR TITLE
hotfix

### PR DESCRIPTION
**Pull request by czarnykapsel**

### CODE REVIEW

**Comment by przemekgrabski** [comment]
Calkiem srednio

File: crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/web/CurrencyNameServlet.java [hunk 1]
```diff
@@ -34,13 +34,13 @@
     @Inject
     private LoginService loginService;
 
-    @Override
+/*    @Override
     public void init() {
```
Comment (isacademyci) [inline crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/web/CurrencyNameServlet.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/service/ConfigurationService.java [hunk 2]
```diff
@@ -16,13 +16,13 @@
     @Inject
     private ConfigurationDao configurationDao;
 
-    @PostConstruct
+  /*  @PostConstruct
     public void insertConfiguration() {
```
Comment (isacademyci) [inline crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/service/ConfigurationService.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/service/LoginService.java [hunk 3]
```diff
@@ -25,11 +25,11 @@
     private static final String SESSION_ATTRIBUTE_EMAIL = "userEmail";
     private static final Integer ADMIN = 1;
 
-    @PostConstruct
+ /*   @PostConstruct
     public void inserUser() {
```
Comment (isacademyci) [inline crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/service/LoginService.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by isacademyci** [comment]
SonarQube analysis reported 4 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [CurrencyNameServlet.java#L70](https://github.com/infoshareacademy/jjdd5-zajavka/blob/2f46a15e3062155ba363b7f7fa89d5b674011c86/crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/web/CurrencyNameServlet.java#L70): Use already-defined constant 'TEMPLATE_NAME' instead of duplicating its value here. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**